### ISSUE TITLE
Deployement failed // docker installation

### ISSUE THREAD

**Issue by SOUIHEL-ayoub** [vmware-tanzu/community-edition#2207]
Hello , 
i am trying to deploy the management cluster on CentOS , but each time the deployment fails , here the deployment log :

[ayoub@localhost ~]$ tanzu management-cluster create -i docker --name mgmtcl -v 10 --plan dev --ceip-participation=false
compatibility file (/home/ayoub/.config/tanzu/tkg/compatibility/tkg-compatibility.yaml) already exists, skipping download
BOM files inside /home/ayoub/.config/tanzu/tkg/bom already exists, skipping download
cluster config file not provided using default config file at '/home/ayoub/.config/tanzu/tkg/cluster-config.yaml'
CEIP Opt-in status: false

Validating the pre-requisites...
Identity Provider not configured. Some authentication features won't work.

Setting up management cluster...
Validating configuration...
Using infrastructure provider docker:v0.3.23
Generating cluster configuration...
Setting up bootstrapper...
Fetching configuration for kind node image...
kindConfig:
 &{{Cluster kind.x-k8s.io/v1alpha4}  [{  map[] [{/var/run/docker.sock /var/run/docker.sock false false }] [] [] []}] { 0  100.96.0.0/11 100.64.0.0/13 false } map[] map[] [apiVersion: kubeadm.k8s.io/v1beta2
kind: ClusterConfiguration
imageRepository: projects.registry.vmware.com/tkg
etcd:
  local:
    imageRepository: projects.registry.vmware.com/tkg
    imageTag: v3.4.13_vmware.15
dns:
  type: CoreDNS
  imageRepository: projects.registry.vmware.com/tkg
  imageTag: v1.8.0_vmware.5] [] [] []}
Creating kind cluster: tkg-kind-c5g9p9n170hk8deakkcg
Creating cluster "tkg-kind-c5g9p9n170hk8deakkcg" ...
Ensuring node image (projects.registry.vmware.com/tkg/kind/node:v1.21.2_vmware.1) ...
Image: projects.registry.vmware.com/tkg/kind/node:v1.21.2_vmware.1 present locally
Preparing nodes ...
Writing configuration ...
Using the following kubeadm config for node tkg-kind-c5g9p9n170hk8deakkcg-control-plane:apiServer: certSANs: - localhost - 127.0.0.1 extraArgs: runtime-config: ""apiVersion: kubeadm.k8s.io/v1beta2clusterName: tkg-kind-c5g9p9n170hk8deakkcgcontrolPlaneEndpoint: tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443controllerManager: extraArgs: enable-hostpath-provisioner: "true"dns: imageRepository: projects.registry.vmware.com/tkg imageTag: v1.8.0_vmware.5 type: CoreDNSetcd: local: imageRepository: projects.registry.vmware.com/tkg imageTag: v3.4.13_vmware.15imageRepository: projects.registry.vmware.com/tkgkind: ClusterConfigurationkubernetesVersion: v1.21.2+vmware.1networking: podSubnet: 100.96.0.0/11 serviceSubnet: 100.64.0.0/13scheduler: extraArgs: null---apiVersion: kubeadm.k8s.io/v1beta2bootstrapTokens:- token: abcdef.0123456789abcdefkind: InitConfigurationlocalAPIEndpoint: advertiseAddress: 172.18.0.3 bindPort: 6443nodeRegistration: criSocket: unix:///run/containerd/containerd.sock kubeletExtraArgs: fail-swap-on: "false" node-ip: 172.18.0.3 node-labels: "" provider-id: kind://docker/tkg-kind-c5g9p9n170hk8deakkcg/tkg-kind-c5g9p9n170hk8deakkcg-control-plane---apiVersion: kubeadm.k8s.io/v1beta2controlPlane: localAPIEndpoint: advertiseAddress: 172.18.0.3 bindPort: 6443discovery: bootstrapToken: apiServerEndpoint: tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443 token: abcdef.0123456789abcdef unsafeSkipCAVerification: truekind: JoinConfigurationnodeRegistration: criSocket: unix:///run/containerd/containerd.sock kubeletExtraArgs: fail-swap-on: "false" node-ip: 172.18.0.3 node-labels: "" provider-id: kind://docker/tkg-kind-c5g9p9n170hk8deakkcg/tkg-kind-c5g9p9n170hk8deakkcg-control-plane---apiVersion: kubelet.config.k8s.io/v1beta1cgroupDriver: cgroupfsevictionHard: imagefs.available: 0% nodefs.available: 0% nodefs.inodesFree: 0%imageGCHighThresholdPercent: 100kind: KubeletConfiguration---apiVersion: kubeproxy.config.k8s.io/v1alpha1conntrack: maxPerCore: 0iptables: minSyncPeriod: 1skind: KubeProxyConfigurationmode: iptables
Starting control-plane ...
I1008 19:31:56.838685 153 initconfiguration.go:246] loading configuration from "/kind/kubeadm.conf"[config] WARNING: Ignored YAML document with GroupVersionKind kubeadm.k8s.io/v1beta2, Kind=JoinConfigurationI1008 19:31:56.926781 153 certs.go:110] creating a new certificate authority for ca[init] Using Kubernetes version: v1.21.2+vmware.1[certs] Using certificateDir folder "/etc/kubernetes/pki"[certs] Generating "ca" certificate and keyI1008 19:31:57.101127 153 certs.go:487] validating certificate period for ca certificate[certs] Generating "apiserver" certificate and key[certs] apiserver serving cert is signed for DNS names [kubernetes kubernetes.default kubernetes.default.svc kubernetes.default.svc.cluster.local localhost tkg-kind-c5g9p9n170hk8deakkcg-control-plane] and IPs [100.64.0.1 172.18.0.3 127.0.0.1][certs] Generating "apiserver-kubelet-client" certificate and keyI1008 19:31:57.991960 153 certs.go:110] creating a new certificate authority for front-proxy-ca[certs] Generating "front-proxy-ca" certificate and keyI1008 19:31:58.403082 153 certs.go:487] validating certificate period for front-proxy-ca certificate[certs] Generating "front-proxy-client" certificate and keyI1008 19:31:58.593609 153 certs.go:110] creating a new certificate authority for etcd-ca[certs] Generating "etcd/ca" certificate and keyI1008 19:31:58.960577 153 certs.go:487] validating certificate period for etcd/ca certificate[certs] Generating "etcd/server" certificate and key[certs] etcd/server serving cert is signed for DNS names [localhost tkg-kind-c5g9p9n170hk8deakkcg-control-plane] and IPs [172.18.0.3 127.0.0.1 ::1][certs] Generating "etcd/peer" certificate and key[certs] etcd/peer serving cert is signed for DNS names [localhost tkg-kind-c5g9p9n170hk8deakkcg-control-plane] and IPs [172.18.0.3 127.0.0.1 ::1][certs] Generating "etcd/healthcheck-client" certificate and keyI1008 19:32:00.092458 153 certs.go:76] creating new public/private key files for signing service account users[certs] Generating "apiserver-etcd-client" certificate and key[certs] Generating "sa" key and public key[kubeconfig] Using kubeconfig folder "/etc/kubernetes"I1008 19:32:00.217392 153 kubeconfig.go:101] creating kubeconfig file for admin.conf[kubeconfig] Writing "admin.conf" kubeconfig fileI1008 19:32:00.771581 153 kubeconfig.go:101] creating kubeconfig file for kubelet.conf[kubeconfig] Writing "kubelet.conf" kubeconfig fileI1008 19:32:01.714487 153 kubeconfig.go:101] creating kubeconfig file for controller-manager.conf[kubeconfig] Writing "controller-manager.conf" kubeconfig fileI1008 19:32:05.160372 153 kubeconfig.go:101] creating kubeconfig file for scheduler.conf[kubeconfig] Writing "scheduler.conf" kubeconfig fileI1008 19:32:06.184230 153 kubelet.go:63] Stopping the kubelet[kubelet-start] Writing kubelet environment file with flags to file "/var/lib/kubelet/kubeadm-flags.env"[kubelet-start] Writing kubelet configuration to file "/var/lib/kubelet/config.yaml"[kubelet-start] Starting the kubelet[control-plane] Using manifest folder "/etc/kubernetes/manifests"[control-plane] Creating static Pod manifest for "kube-apiserver"I1008 19:32:10.417461 153 manifests.go:96] [control-plane] getting StaticPodSpecsI1008 19:32:10.418055 153 certs.go:487] validating certificate period for CA certificateI1008 19:32:10.418141 153 manifests.go:109] [control-plane] adding volume "ca-certs" for component "kube-apiserver"I1008 19:32:10.418148 153 manifests.go:109] [control-plane] adding volume "etc-ca-certificates" for component "kube-apiserver"I1008 19:32:10.418152 153 manifests.go:109] [control-plane] adding volume "k8s-certs" for component "kube-apiserver"I1008 19:32:10.418156 153 manifests.go:109] [control-plane] adding volume "usr-local-share-ca-certificates" for component "kube-apiserver"I1008 19:32:10.418161 153 manifests.go:109] [control-plane] adding volume "usr-share-ca-certificates" for component "kube-apiserver"I1008 19:32:10.451200 153 manifests.go:126] [control-plane] wrote static Pod manifest for component "kube-apiserver" to "/etc/kubernetes/manifests/kube-apiserver.yaml"I1008 19:32:10.451228 153 manifests.go:96] [control-plane] getting StaticPodSpecsI1008 19:32:10.451605 153 manifests.go:109] [control-plane] adding volume "ca-certs" for component "kube-controller-manager"I1008 19:32:10.451614 153 manifests.go:109] [control-plane] adding volume "etc-ca-certificates" for component "kube-controller-manager"I1008 19:32:10.451618 153 manifests.go:109] [control-plane] adding volume "flexvolume-dir" for component "kube-controller-manager"I1008 19:32:10.451623 153 manifests.go:109] [control-plane] adding volume "k8s-certs" for component "kube-controller-manager"I1008 19:32:10.451627 153 manifests.go:109] [control-plane] adding volume "kubeconfig" for component "kube-controller-manager"I1008 19:32:10.451631 153 manifests.go:109] [control-plane] adding volume "usr-local-share-ca-certificates" for component "kube-controller-manager"I1008 19:32:10.451635 153 manifests.go:109] [control-plane] adding volume "usr-share-ca-certificates" for component "kube-controller-manager"[control-plane] Creating static Pod manifest for "kube-controller-manager"I1008 19:32:10.462734 153 manifests.go:126] [control-plane] wrote static Pod manifest for component "kube-controller-manager" to "/etc/kubernetes/manifests/kube-controller-manager.yaml"I1008 19:32:10.462825 153 manifests.go:96] [control-plane] getting StaticPodSpecsI1008 19:32:10.463533 153 manifests.go:109] [control-plane] adding volume "kubeconfig" for component "kube-scheduler"I1008 19:32:10.464389 153 manifests.go:126] [control-plane] wrote static Pod manifest for component "kube-scheduler" to "/etc/kubernetes/manifests/kube-scheduler.yaml"I1008 19:32:10.465453 153 local.go:74] [etcd] wrote Static Pod manifest for a local etcd member to "/etc/kubernetes/manifests/etcd.yaml"I1008 19:32:10.465476 153 waitcontrolplane.go:87] [wait-control-plane] Waiting for the API server to be healthyI1008 19:32:10.466431 153 loader.go:372] Config loaded from file: /etc/kubernetes/admin.conf[control-plane] Creating static Pod manifest for "kube-scheduler"[etcd] Creating static Pod manifest for local etcd in "/etc/kubernetes/manifests"[wait-control-plane] Waiting for the kubelet to boot up the control plane as static Pods from directory "/etc/kubernetes/manifests". This can take up to 4m0sI1008 19:32:10.473831 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 6 millisecondsI1008 19:32:10.987598 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 4 millisecondsI1008 19:32:11.723708 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 11 millisecondsI1008 19:32:11.996959 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 13 millisecondsI1008 19:32:12.492691 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 5 millisecondsI1008 19:32:13.011323 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 6 millisecondsI1008 19:32:13.593707 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 118 millisecondsI1008 19:32:13.988898 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 12 millisecondsI1008 19:32:14.580781 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 95 millisecondsI1008 19:32:15.121141 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 43 millisecondsI1008 19:32:16.223025 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 458 millisecondsI1008 19:32:16.490256 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 3 millisecondsI1008 19:32:17.055743 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 3 millisecondsI1008 19:32:19.402782 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 1304 millisecondsI1008 19:32:19.574360 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 99 millisecondsI1008 19:32:20.585787 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 582 millisecondsI1008 19:32:20.648798 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 62 millisecondsI1008 19:32:21.357067 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 243 millisecondsI1008 19:32:21.717343 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 233 millisecondsI1008 19:32:22.108310 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 132 millisecondsI1008 19:32:23.025698 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 547 millisecondsI1008 19:32:23.630422 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 7 millisecondsI1008 19:32:24.294269 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 246 millisecondsI1008 19:32:24.695937 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 147 millisecondsI1008 19:32:25.149370 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 76 millisecondsI1008 19:32:25.618934 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 96 millisecondsI1008 19:32:25.980047 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 2 millisecondsI1008 19:32:26.483046 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 8 millisecondsI1008 19:32:26.975611 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 1 millisecondsI1008 19:32:27.545198 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 55 millisecondsI1008 19:32:28.117851 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 137 millisecondsI1008 19:32:28.965841 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 239 millisecondsI1008 19:32:29.077878 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 67 millisecondsI1008 19:32:29.478977 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 3 millisecondsI1008 19:32:30.092614 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 49 millisecondsI1008 19:32:30.515414 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 3 millisecondsI1008 19:32:31.049509 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 33 millisecondsI1008 19:32:31.485785 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 7 millisecondsI1008 19:32:32.020992 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 16 millisecondsI1008 19:32:32.746627 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 5 millisecondsI1008 19:32:32.982224 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 4 millisecondsI1008 19:32:33.478241 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 2 millisecondsI1008 19:32:34.002765 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 7 millisecondsI1008 19:32:34.482143 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 4 millisecondsI1008 19:32:34.979152 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 3 millisecondsI1008 19:32:35.478739 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 4 millisecondsI1008 19:32:35.984940 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 10 millisecondsI1008 19:32:36.483026 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 6 millisecondsI1008 19:32:36.982211 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 2 millisecondsI1008 19:32:37.477076 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 2 millisecondsI1008 19:32:37.976913 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 2 millisecondsI1008 19:32:38.479052 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 2 millisecondsI1008 19:32:38.977055 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 2 millisecondsI1008 19:32:39.477791 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s in 3 millisecondsI1008 19:32:46.117880 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 6142 millisecondsI1008 19:32:46.514456 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 38 millisecondsI1008 19:32:46.989304 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 6 millisecondsI1008 19:32:47.537954 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 30 millisecondsI1008 19:32:48.109688 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 133 millisecondsI1008 19:32:48.589484 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 94 millisecondsI1008 19:32:49.011757 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 12 millisecondsI1008 19:32:49.505384 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 28 millisecondsI1008 19:32:50.050628 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 74 milliseconds[kubelet-check] Initial timeout of 40s passed.I1008 19:32:50.576661 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 94 millisecondsI1008 19:32:51.195932 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 196 millisecondsI1008 19:32:52.485524 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 792 millisecondsI1008 19:32:53.076627 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 53 millisecondsI1008 19:32:53.796312 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 321 millisecondsI1008 19:32:56.379148 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 2066 millisecondsI1008 19:32:58.747912 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 2159 millisecondsI1008 19:32:59.341738 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 367 millisecondsI1008 19:32:59.482531 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 6 millisecondsI1008 19:33:00.571869 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 56 millisecondsI1008 19:33:01.288920 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 298 millisecondsI1008 19:33:02.134347 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 659 millisecondsI1008 19:33:02.873293 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 396 millisecondsI1008 19:33:03.187749 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 77 millisecondsI1008 19:33:03.694021 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 28 millisecondsI1008 19:33:04.354034 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 289 millisecondsI1008 19:33:06.441700 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 1966 millisecondsI1008 19:33:06.616944 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 500 Internal Server Error in 140 millisecondsI1008 19:33:06.987790 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/healthz?timeout=10s 200 OK in 11 millisecondsI1008 19:33:06.988031 153 uploadconfig.go:108] [upload-config] Uploading the kubeadm ClusterConfiguration to a ConfigMap[apiclient] All control plane components are healthy after 56.520557 seconds[upload-config] Storing the configuration used in ConfigMap "kubeadm-config" in the "kube-system" NamespaceI1008 19:33:07.005882 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/configmaps?timeout=10s 201 Created in 12 millisecondsI1008 19:33:07.014394 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/namespaces/kube-system/roles?timeout=10s 201 Created in 7 millisecondsI1008 19:33:07.020768 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/namespaces/kube-system/rolebindings?timeout=10s 201 Created in 4 millisecondsI1008 19:33:07.021411 153 uploadconfig.go:122] [upload-config] Uploading the kubelet component config to a ConfigMap[kubelet] Creating a ConfigMap "kubelet-config-1.21" in namespace kube-system with the configuration for the kubelets in the clusterI1008 19:33:07.026530 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/configmaps?timeout=10s 201 Created in 3 millisecondsI1008 19:33:07.039340 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/namespaces/kube-system/roles?timeout=10s 201 Created in 12 millisecondsI1008 19:33:07.057768 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/namespaces/kube-system/rolebindings?timeout=10s 201 Created in 17 millisecondsI1008 19:33:07.057895 153 uploadconfig.go:127] [upload-config] Preserving the CRISocket information for the control-plane nodeI1008 19:33:07.057919 153 patchnode.go:30] [patchnode] Uploading the CRI Socket information "unix:///run/containerd/containerd.sock" to the Node API object "tkg-kind-c5g9p9n170hk8deakkcg-control-plane" as an annotationI1008 19:33:07.568801 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/nodes/tkg-kind-c5g9p9n170hk8deakkcg-control-plane?timeout=10s 200 OK in 9 millisecondsI1008 19:33:07.580752 153 round_trippers.go:454] PATCH https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/nodes/tkg-kind-c5g9p9n170hk8deakkcg-control-plane?timeout=10s 200 OK in 8 milliseconds[upload-certs] Skipping phase. Please see --upload-certs[mark-control-plane] Marking the node tkg-kind-c5g9p9n170hk8deakkcg-control-plane as control-plane by adding the labels: [node-role.kubernetes.io/master(deprecated) node-role.kubernetes.io/control-plane node.kubernetes.io/exclude-from-external-load-balancers][mark-control-plane] Marking the node tkg-kind-c5g9p9n170hk8deakkcg-control-plane as control-plane by adding the taints [node-role.kubernetes.io/master:NoSchedule]I1008 19:33:08.105746 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/nodes/tkg-kind-c5g9p9n170hk8deakkcg-control-plane?timeout=10s 200 OK in 9 millisecondsI1008 19:33:08.123680 153 round_trippers.go:454] PATCH https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/nodes/tkg-kind-c5g9p9n170hk8deakkcg-control-plane?timeout=10s 200 OK in 16 milliseconds[bootstrap-token] Configuring bootstrap tokens, cluster-info ConfigMap, RBAC RolesI1008 19:33:08.129958 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/secrets/bootstrap-token-abcdef?timeout=10s 404 Not Found in 5 millisecondsI1008 19:33:08.140826 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/secrets?timeout=10s 201 Created in 10 milliseconds[bootstrap-token] configured RBAC rules to allow Node Bootstrap tokens to get nodesI1008 19:33:08.214817 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/clusterroles?timeout=10s 201 Created in 64 millisecondsI1008 19:33:08.235271 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings?timeout=10s 201 Created in 19 milliseconds[bootstrap-token] configured RBAC rules to allow Node Bootstrap tokens to post CSRs in order for nodes to get long term certificate credentialsI1008 19:33:08.249392 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings?timeout=10s 201 Created in 13 milliseconds[bootstrap-token] configured RBAC rules to allow the csrapprover controller automatically approve CSRs from a Node Bootstrap TokenI1008 19:33:08.254452 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings?timeout=10s 201 Created in 4 milliseconds[bootstrap-token] configured RBAC rules to allow certificate rotation for all node client certificates in the clusterI1008 19:33:08.261230 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings?timeout=10s 201 Created in 6 millisecondsI1008 19:33:08.261364 153 clusterinfo.go:45] [bootstrap-token] loading admin kubeconfigI1008 19:33:08.261920 153 loader.go:372] Config loaded from file: /etc/kubernetes/admin.confI1008 19:33:08.261938 153 clusterinfo.go:56] [bootstrap-token] copying the cluster from admin.conf to the bootstrap kubeconfigI1008 19:33:08.262232 153 clusterinfo.go:68] [bootstrap-token] creating/updating ConfigMap in kube-public namespace[bootstrap-token] Creating the "cluster-info" ConfigMap in the "kube-public" namespaceI1008 19:33:08.275807 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-public/configmaps?timeout=10s 201 Created in 13 millisecondsI1008 19:33:08.275947 153 clusterinfo.go:82] creating the RBAC rules for exposing the cluster-info ConfigMap in the kube-public namespaceI1008 19:33:08.281797 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/namespaces/kube-public/roles?timeout=10s 201 Created in 5 millisecondsI1008 19:33:08.296610 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/namespaces/kube-public/rolebindings?timeout=10s 201 Created in 14 millisecondsI1008 19:33:08.296786 153 kubeletfinalize.go:88] [kubelet-finalize] Assuming that kubelet client certificate rotation is enabled: found "/var/lib/kubelet/pki/kubelet-client-current.pem"I1008 19:33:08.297288 153 loader.go:372] Config loaded from file: /etc/kubernetes/kubelet.conf[kubelet-finalize] Updating "/etc/kubernetes/kubelet.conf" to point to a rotatable kubelet client certificate and keyI1008 19:33:08.304594 153 kubeletfinalize.go:132] [kubelet-finalize] Restarting the kubelet to enable client certificate rotationI1008 19:33:09.572419 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/apps/v1/namespaces/kube-system/deployments?labelSelector=k8s-app%3Dkube-dns 200 OK in 5 millisecondsI1008 19:33:09.583834 153 round_trippers.go:454] GET https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/configmaps/coredns?timeout=10s 404 Not Found in 4 millisecondsI1008 19:33:09.596894 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/configmaps?timeout=10s 201 Created in 12 millisecondsI1008 19:33:09.601653 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/clusterroles?timeout=10s 201 Created in 4 millisecondsI1008 19:33:09.613615 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings?timeout=10s 201 Created in 11 millisecondsI1008 19:33:09.619286 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/serviceaccounts?timeout=10s 201 Created in 4 millisecondsI1008 19:33:09.699725 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/apps/v1/namespaces/kube-system/deployments?timeout=10s 201 Created in 64 millisecondsI1008 19:33:09.715472 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/services?timeout=10s 201 Created in 13 milliseconds[addons] Applied essential addon: CoreDNSI1008 19:33:09.727982 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/serviceaccounts?timeout=10s 201 Created in 12 millisecondsI1008 19:33:09.741345 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/api/v1/namespaces/kube-system/configmaps?timeout=10s 201 Created in 11 millisecondsI1008 19:33:09.775833 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/apps/v1/namespaces/kube-system/daemonsets?timeout=10s 201 Created in 28 millisecondsI1008 19:33:09.791165 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings?timeout=10s 201 Created in 15 millisecondsI1008 19:33:09.814127 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/namespaces/kube-system/roles?timeout=10s 201 Created in 22 millisecondsI1008 19:33:10.033029 153 round_trippers.go:454] POST https://tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443/apis/rbac.authorization.k8s.io/v1/namespaces/kube-system/rolebindings?timeout=10s 201 Created in 218 millisecondsI1008 19:33:10.034068 153 loader.go:372] Config loaded from file: /etc/kubernetes/admin.conf[addons] Applied essential addon: kube-proxyI1008 19:33:10.042603 153 loader.go:372] Config loaded from file: /etc/kubernetes/admin.confYour Kubernetes control-plane has initialized successfully!To start using your cluster, you need to run the following as a regular user: mkdir -p $HOME/.kube sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config sudo chown $(id -u):$(id -g) $HOME/.kube/configAlternatively, if you are the root user, you can run: export KUBECONFIG=/etc/kubernetes/admin.confYou should now deploy a pod network to the cluster.Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at: https://kubernetes.io/docs/concepts/cluster-administration/addons/You can now join any number of control-plane nodes by copying certificate authoritiesand service account keys on each node and then running the following as root: kubeadm join tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443 --token <value withheld> \--discovery-token-ca-cert-hash sha256:062478b376c9f153179bfdd6604a4a87df0e9adfaa82dd36e356cfcb6fc7ea32 \--control-plane Then you can join any number of worker nodes by running the following on each as root:kubeadm join tkg-kind-c5g9p9n170hk8deakkcg-control-plane:6443 --token <value withheld> \--discovery-token-ca-cert-hash sha256:062478b376c9f153179bfdd6604a4a87df0e9adfaa82dd36e356cfcb6fc7ea32
Installing CNI ...
Installing StorageClass ...
Waiting 2m0s for control-plane = Ready ...
Ready after 1m42s
Bootstrapper created. Kubeconfig: /home/ayoub/.kube-tkg/tmp/config_AlCWPzuy
Installing providers on bootstrapper...
Fetching providers
Installing cert-manager Version="v1.1.0"
Waiting for cert-manager to be available...
Installing Provider="cluster-api" Version="v0.3.23" TargetNamespace="capi-system"
Installing Provider="bootstrap-kubeadm" Version="v0.3.23" TargetNamespace="capi-kubeadm-bootstrap-system"
Installing Provider="control-plane-kubeadm" Version="v0.3.23" TargetNamespace="capi-kubeadm-control-plane-system"
Installing Provider="infrastructure-docker" Version="v0.3.23" TargetNamespace="capd-system"
installed  Component=="cluster-api"  Type=="CoreProvider"  Version=="v0.3.23"
installed  Component=="kubeadm"  Type=="BootstrapProvider"  Version=="v0.3.23"
installed  Component=="kubeadm"  Type=="ControlPlaneProvider"  Version=="v0.3.23"
installed  Component=="docker"  Type=="InfrastructureProvider"  Version=="v0.3.23"
Waiting for provider cluster-api
Waiting for provider infrastructure-docker
Waiting for provider bootstrap-kubeadm
Waiting for provider control-plane-kubeadm
Waiting for resource capd-controller-manager of type *v1.Deployment to be up and running
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
Waiting for resource capi-kubeadm-control-plane-controller-manager of type *v1.Deployment to be up and running
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
Waiting for resource capi-kubeadm-bootstrap-controller-manager of type *v1.Deployment to be up and running
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
Waiting for resource capi-controller-manager of type *v1.Deployment to be up and running
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
Waiting for resource capi-controller-manager of type *v1.Deployment to be up and running
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying
pods are not yet running for deployment 'capi-kubeadm-bootstrap-controller-manager' in namespace 'capi-kubeadm-bootstrap-system', retrying
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
Waiting for resource capi-kubeadm-control-plane-controller-manager of type *v1.Deployment to be up and running
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-webhook-system', retrying
Waiting for resource capi-kubeadm-bootstrap-controller-manager of type *v1.Deployment to be up and running
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
Passed waiting on provider bootstrap-kubeadm after 1m50.915413542s
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
Passed waiting on provider control-plane-kubeadm after 1m55.723343089s
pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
Passed waiting on provider cluster-api after 2m0.863713651s
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
Passed waiting on provider infrastructure-docker after 2m55.384854993s
Success waiting on all providers.
Start creating management cluster...
patch cluster object with operation status:
        {
                "metadata": {
                        "annotations": {
                                "TKGOperationInfo" : "{\"Operation\":\"Create\",\"OperationStartTimestamp\":\"2021-10-08 19:42:28.558113026 +0000 UTC\",\"OperationTimeout\":1800}",
                                "TKGOperationLastObservedTimestamp" : "2021-10-08 19:42:28.558113026 +0000 UTC"
                        }
                }
        }

deployment failed with the below error 

pods are not yet running for deployment 'capi-controller-manager' in namespace 'capi-webhook-system', retrying
pods are not yet running for deployment 'capd-controller-manager' in namespace 'capd-system', retrying
pods are not yet running for deployment 'capi-kubeadm-control-plane-controller-manager' in namespace 'capi-kubeadm-control-plane-system', retrying

**Comment by stmcginnis** [comment]
Thanks! The `docker ps` output is a little odd. It looks like there are two bootstrap clusters running, and the mgmtcl-control-plane container is one of the expected management cluster containers, but there should have been an HAProxy container created first.

I wonder if you could try cleaning up a bit and trying again. If these are all that are running, please try:

```
docker rm -f $(docker ps -a -a)
docker system prune -a --volumes
```

Then run the `tanzu management-cluster create` command again.

One other thing that comes to mind. I haven't tested with CentOS myself, but it should be fine. There is a [known issue with cgroupsv2](https://github.com/vmware-tanzu/community-edition/issues/2138#issuecomment-938038394) though. One other thing to check would be to run `grep cgroup /proc/filesystems` and see if it's v1 or v2.

**Comment by stmcginnis** [comment]
Hey @SOUIHEL-ayoub. Just going through and following up on open issues. Sorry I didn't check back sooner, but just wondering if you were ever able to get by this issue after running the docker cleanup commands I posted? Please let me know if you are still having problems with this, or if we can just close this out. Thanks!

@rajks24 - I guess same for you if you were able to run those commands. If you were not able to get by that, please open a new issue to track your problem, or hit us up on Slack and we can try to troubleshoot some more.

For both of you, please note standalone-cluster is being deprecated in the next release and we are introducing a new `unamanged-cluster` option that is much more efficient for creating local clusters using docker.

**Comment by stmcginnis** [comment]
I'm going to assume this is working now, but feel free to reopen if there are still issues.
